xc7/tests/litex/minilitex/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_file_target(FILE mem.init)
```

Comment:
@litghost These files are imported in the verilog file. Hence there need to be links to it in the minitest's build directory.
What is the best way to add the file targets as dependencies to the fpga target so that the links are created. Currently I have to call make file_mem_init... explicitly to produce the links.

Reply:
Imports of this style should likely be detected by https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/utils/deps_verilog.py

Can you modify https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/utils/deps_verilog.py to output them, if they are not already?

Reply:
Turned out to be a matter of a single letter. Now it works perfectly.

Reply:
What letter was missing?

Reply:
`h` see [here](https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1181/commits/fc8217c620926f141fb2e6ff3ef2faaa851d5b52). We have readmemh in the litex design.